src/tests/stdlib/strncat.c

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file
+ *
+ * @date Mar 31, 2020
+ * @author: Prajawal Kumar Pandey
+ */
+
+#include <string.h>
+#include <stdlib.h>
+#include <embox/test.h>
+
+EMBOX_TEST_SUITE("test suite for strncat()");
+
+TEST_CASE("concatenate full string with strncat()") {
+    char src[] = "string";
+    char dst[] = "char ";
+
+    strncat(dst, src, sizeof(src));
+    test_assert_not_null(dst);
+    test_assert_zero(strcmp(dst, "char string"));
+}
+
+TEST_CASE("concatenate empty string with strncat()") {
+    char src[] = "";
+    char dst[] = "char ";
+
+    strncat(dst, src, sizeof(src));
+    test_assert_not_null(dst);
+    test_assert_zero(strcmp(dst, "char "));
+}
+
+TEST_CASE("concatenate to an empty string with strncat()") {
+    char src[] = "string";
+    char dst[] = "";
+
+    strncat(dst, src, sizeof(src));
+    test_assert_not_null(dst);
+    test_assert_zero(strcmp(dst, src));
+}
+
+TEST_CASE("concatenate part of a string with strncat()") {
+    char src[] = "string";
+    const int prefix_len = 3;
```

Comment:
why did you declare variables only for passing into `strncat(dst, src, prefix_len);`?

Reply:
> why did you declare variables

Updated!

Reply:
Updated!